Automation Backfill: Issue 40385: Can't Update Samples Using a File

labkey-danield opened the pull request:
#### Rationale
Issue 40385: Can't Update Samples Using a File

#### Related Pull Requests
* None

#### Changes
* Added automation for Issue 40385: Can't Update Samples Using a File
* Removed old custom logging code, replaced with calls to checker().
* Some general clean-up, make test more readable (hopefully).

labkey-danield committed:
Merge branch 'develop' into fb_smAutomationBackfill

labkey-danield committed:
Test for Issue 40385: Can't Update Samples Using a File

labkey-danield committed:
Merge branch 'develop' into fb_smAutomationBackfill

labkey-danield committed:
Merge branch 'develop' into fb_smAutomationBackfill

labkey-susanh commented on src/org/labkey/test/tests/SampleTypeTest.java:
```diff
@@ -713,32 +698,47 @@ public void testUpdateAndDeleteWithCommentsAndFlags()
 
-        errorMsg = "Sample Type data is not as expected after deleting the Description and Flag/Comment.";
-        if(!checkExpectedAgainstDB(sampleData, errorMsg))
+        resultsFromDB = getSampleDataFromDB("/" + PROJECT_NAME,SAMPLE_TYPE_NAME, dbFieldsToCheck);
```
Would `getCurrentContainerPath()` return this catenated projectName and "/"?

labkey-danield replied:
Yes it does.
Removed all the `"/" + PROJECT_NAME` and replaced with `getCurrentContainerPath()`.

labkey-danield committed:
Merge branch 'develop' into fb_smAutomationBackfill

labkey-danield committed:
Implementing code review feedback.
Removed all the "/" + PROJECT_NAME and replaced with getCurrentContainerPath().
